### PR TITLE
fix notification request(headers and data)

### PR DESCRIPTION
**Pull request by SarunasAzna**

### CODE REVIEW

File: src/containers/Notifications.js [hunk 1]
```diff
@@ -32,8 +32,13 @@ class Notifications extends Component {
 	handleSubmit(event) {
 		event.preventDefault();
 
-		axios
-			.put(`${config.API_BASE_URL}/users/me/`)
+		axios({
+            method: 'put',
```
Comment (SarunasAzna) [inline src/containers/Notifications.js]: should i use tabs instead of spaces here? it looks different on my editor when in github...

Comment (superkarolis) [reply src/containers/Notifications.js]: I use tabs personally

Comment (BeOleg) [reply src/containers/Notifications.js]: Is there a generic way to set headers?
https://github.com/axios/axios/issues/209
https://stackoverflow.com/questions/43051291/attach-authorization-header-for-all-axios-requests

Comment (superkarolis) [reply src/containers/Notifications.js]: I could work on this in a separate PR.

**Pull request merged**